pkg/sql/sem/builtins/fingerprint_builtin_test.go

```diff
@@ -217,3 +218,58 @@ func TestFingerprint(t *testing.T) {
 
 	require.Equal(t, fingerprintPreSplit, fingerprintPostSplit)
 }
+
+func TestFingerprintConcurrentTransactions(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	ctx := context.Background()
+	s, sqlDB, _ := serverutils.StartServer(t, base.TestServerArgs{})
+	defer s.Stopper().Stop(ctx)
+	db := sqlutils.MakeSQLRunner(sqlDB)
+	db.Exec(t, "CREATE DATABASE IF NOT EXISTS test")
+	db.Exec(t, "CREATE TABLE test.test (k PRIMARY KEY) AS SELECT generate_series(1, 10)")
+
+	startTs := s.Clock().Now()
+
+	fingerprintQuery := `
+SELECT *
+FROM
+	crdb_internal.fingerprint(
+		crdb_internal.table_span((SELECT id FROM system.namespace WHERE name = 'test' AND "parentID" != 0)),
```

Comment:
no need to change but I _think_ you can also do 'test.test'::REGCLASS and it would give you the descriptor ID. It does a namespace lookup as part of the cast I believe.

Reply:
Oh interesting TIL:

```
select crdb_internal.table_span('movr.users'::REGCLASS::INT);                                                                                                                                     
  crdb_internal.table_span
-----------------------------
  {"\\xfe8af2","\\xfe8af3"}
(1 row)
```

Reply:
Going to leave as-is since CI is green, but that is a very useful trick, thanks.